core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-create/container-create.component.html

```diff
@@ -2,13 +2,15 @@
     <div class="container-create__tab-container">
```

Comment:
![image](https://user-images.githubusercontent.com/37185433/193038171-8b69730a-bee9-4bd4-a499-184192cf564f.png)

missing the "dot-" prefix

Reply:
@alfredo-dotcms , Fixed.